package.json

```diff
@@ -19,20 +19,22 @@
     "eslint": "eslint lib test --ext .ts --ignore-pattern *.d.ts",
     "lint-md": "remark -u preset-lint-recommended .",
     "lint": "npm run lint-md && npm run eslint",
-    "test": "mocha --require ts-node/register --require source-map-support/register --full-trace test/*.ts",
-    "test-coverage": "nyc npm run test",
-    "send-coveralls": "nyc report --reporter=text-lcov | coveralls",
+    "test": "mocha",
+    "test-coverage": "c8 npm run test",
+    "send-coveralls": "c8 report --reporter=text-lcov | coveralls",
     "start": "npm run compile && npm run lint && npm run cover-test"
   },
   "engines": {
-    "node": ">=8"
+    "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Borewit/peek-readable"
   },
   "license": "MIT",
   "main": "lib/index.js",
+  "type": "module",
+  "exports": "./lib/index.js",
   "types": "lib/index.d.ts",
   "bugs": {
     "url": "https://github.com/Borewit/peek-readable/issues"
```

Comment:
Unrelated, but this property is moot. It's automatically inferred from the `repository` property.